### PR TITLE
Add to PCJR and Grapple Tutorial 2, 3

### PR DESCRIPTION
**Pull request by kjbranch**
- Clean up various strats and add wall jumps throughout.
- Add canMellaIceClip to PCJR

### CODE REVIEW

File: region/norfair/crocomire.json [hunk 1]
```diff
@@ -1621,6 +1622,19 @@
...
                   ]
+                },
+                {
+                  "name": "Indy Jones Mella Ice Clip",
+                  "notable": true,
+                  "requires": [
+                    "Morph",
+                    "canMellaIceClip",
+                    "canWalljump",
+                    "canCrumbleSpinJump"
```
Comment (osse101) [inline region/norfair/crocomire.json:1633]: There are ways around the crumble jump.  Falling on the mella and jumping again, refreezing the mella above the crumble, using xray standup.  I don't think it should be a requirement.

File: region/norfair/crocomire.json [hunk 2]
```diff
@@ -1621,6 +1622,19 @@
                     "Obstacle B is present here because you need access to node 2 to bring a Mella over.",
                     "Then you need to manipulate it to go high enough to be used as a stepping stone once frozen."
                   ]
+                },
+                {
+                  "name": "Indy Jones Mella Ice Clip",
```
Comment (osse101) [inline region/norfair/crocomire.json:1627]: Mellas take a vague number of hits to position right in my experience.  There could be ways to dodge better but I don't feel that its avoidable.

File: tech.json [hunk 3]
```diff
@@ -536,6 +536,13 @@
                             "canNonTrivialIceClip"
                           ],
                           "note": "Using a frozen Multiviola to clip through a one-tile-high wall."
+                        },
+                        {
+                          "name": "canMellaIceClip",
+                          "requires": [
+                            "canNonTrivialIceClip"
```
Comment (osse101) [inline tech.json:543]: canManipulateMellas should be here as it is described.  But you may be able to dumbly position them ok.

File: region/norfair/crocomire.json (hunk 1 above)
Comment (kjbranch) [reply region/norfair/crocomire.json]: Thanks, I removed this.

File: tech.json (hunk 3 above)
Comment (kjbranch) [reply tech.json:543]: I don't think canManipulateMellas belongs here. That is a method for manipulating them so their peak height increases over time - used to get to the item without the ice clip or to the left door of this same room. The vertical positioning you need to freeze the Mella at is within the range they move initially. Without it is also more in line with the other ice clip techs.

We found out about this trick through sapphronxd and BuggMann: They couldn't figure out how to manipulate the Mellas, so they instead tried to see if the ice clip was possible.

Comment (osse101) [reply tech.json:543]: That's right, the original clip kinda settles this then.

**Pull request merged**